requirements.txt

```diff
@@ -5,3 +5,4 @@ cython>=0.23
 coverage>=4.0
 six>=1.10
 wheel>=0.29
+lxml>=3.6.0
```

Comment:
Good point. I'll move this into the sanity test Dockerfile (or does it need to be in a wider set?)